### DISCUSSION TITLE
Make Spacedrive Default File Browser/Launch at startup previous session

### DISCUSSION THREAD

**Discussion by ketonik** [spacedriveapp/spacedrive#2402] [Feature Request]
Self titled.
Spacedrive, the way it's going, development wise, i'm gona want this as my:
1- Default File Browser in Windowz
2- I'm gona want to have it launch at Start up
3- It's gona need to remember previous session, that is, all the previous windows and window position with all the tabs prior to last shutdown of Windoze.

At the moment i'll have to keep struggling away with the 3rd party file manager 'Files', until the day Spacedrive has disclosure triangles in List View the same as macOS Finder, so hopefully all these elements link up around the same time.

**Comment by utkubakir** [comment]
We noted this down, will work on it sometime in the future :)

**Comment by ohhsodead** [comment]
+1
